valuesets/ValueSet-UKCore-SourceOfServiceRequest.xml

```diff
@@ -1,5 +1,5 @@
 <ValueSet xmlns="http://hl7.org/fhir">
-	<id value="UKCore-ServiceRequestMethod"/>
+	<id value="UKCore-SourceOfServiceRequest"/>
```

Comment:
this might be a breaking change for BaRS - you'd hope it was using url, or name rather than id, but probably best to check with Naminder